contracts/on-chain-claim/src/contract.rs

```diff
@@ -0,0 +1,113 @@
+#![no_std]
+#![allow(unused_attributes)]
+
+pub use address_info::AddressInfo;
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub mod address_info;
+pub mod config;
+
+use crate::config::{SFT_AMOUNT, MAX_REPAIR_GAP};
+
+/// An empty contract. To be used as a template when starting a new contract from scratch.
+#[multiversx_sc::contract]
```

Comment:
Format the code. Shift + Option + F on Mac or Ctrl + Shift + I on Linux. It will be very helpful in the future, for any subsequent PRs, to just focus on the code.